external/fv3fit/tests/emulation/test_data_io.py

```diff
@@ -19,13 +20,40 @@ def test_get_nc_files():
         }
     )
 
+
+def test_get_nc_files():
+
+    xr_dataset = _get_ds()
+
     with tempfile.TemporaryDirectory() as tmpdir:
         num_files = 3
-        paths = [os.path.join(tmpdir, f"file{i}.nc") for i in range(num_files)]
-        for path in paths:
+        orig_paths = [os.path.join(tmpdir, f"file{i}.nc") for i in range(num_files)]
+        for path in orig_paths:
             xr_dataset.to_netcdf(path)
 
         result_files = io.get_nc_files(tmpdir)
         assert len(result_files) == num_files
-        for path in paths:
-            assert path in result_files
+        for path in result_files:
+            assert path in orig_paths
+
+
+class MockGCSFilesystem:
+
+    protocol = ("gs", "gcs")
+
+    def glob(*args):
+        return [
+            "fake-bucket/file1.nc",
+            "fake-bucket/file2.nc",
+        ]
+
+
+@pytest.mark.network
```

Comment:
since you've mocked the filesystem, does this really need network access?

Reply:
Nope! I forgot to delete this after I switched to a mock.